Gemfile

```diff
@@ -118,9 +117,6 @@ group :development, :ci, :test do
 end
 
 
-gem 'nokogiri', '~> 1.13.11', require: false, git:"https://github.com/commitchange/nokogiri.git", tag: "v1.13.11"
```

Comment:
We were on a backport since the current nokogiri didn't work on Ruby 2

Reply:
This is huge!